### DISCUSSION TITLE
gtsave function with actionButtons in R shiny application

### DISCUSSION THREAD

**Discussion by kalsem** [rstudio/gt#1843] [Q&A]
Can someone help me to used gtsave function to save output as pdf file by clicking on action button in r shiny ?

**Comment by obsaditelnost** [comment]
@kalsem what did you try so far? What is your result and what do you expect?

**Reply by kalsem** [reply]
I tried to save the output TFL into PDF file using gtsave function but when i used outside summary function, it is not working.. Also trying to implement functionality to save summary function output on action button click as pdf file.. if you can help ..

**Reply by obsaditelnost** [reply]
Please provide a reproducible example ("code"), otherwise it's unlikely that you'll get an answer. Without that we can only refer to the reference: [actionButton](https://shiny.posit.co/r/articles/build/action-buttons/index.html) and [gtsave](https://gt.rstudio.com/reference/gtsave.html)
